pkgs/development/python-modules/incremental/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, pkgs }:
+
+pkgs.python27Packages.buildPythonPackage rec {
```

Comment:
Don't pass in `pkgs`, nor `pythonpackages`. Instead, have the packages as parameters.